searchscript/search_names.js

```diff
@@ -0,0 +1,12 @@
+xpath = "//a[@class='app-aware-link feed-shared-actor__container-link relative display-flex flex-grow-1']/@href";
```

Comment:
this xpath is too concrete - make it more idiotproof

Reply:
Fixed, now contain three situations and more flexible implementation